src/Compiler/Microsoft.CodeAnalysis.Razor.Compiler/src/Language/Legacy/LegacyCSharpTokenizer.cs

```diff
@@ -15,7 +15,7 @@
 
 namespace Microsoft.AspNetCore.Razor.Language.Legacy;
 
-internal class CSharpTokenizer : Tokenizer
+internal class LegacyCSharpTokenizer : Tokenizer
```

Comment:
We have a lot of "legacy" in the razor codebase, consider adding a comment explaining what it means here.